### ISSUE TITLE
Clarify the Google Analytics UTM codes that are supported on Matomo 

### ISSUE THREAD

**Issue by IEEEPaulGerhard** [piwik/piwik#21955]
This page: 

https://matomo.org/faq/general/faq_119/

Provides context as to what UTM codes are supported by matomo. 

The description text is confusing as to what is supported: 
utm_campaign , utm_source , utm_medium , utm_term , utm_content , utm_id

See description: By default, Matomo offers two dimensions for Campaign tracking: name, and keyword. The parameters utm_campaign, or utm_medium, or utm_source are used as the Campaign name. The parameter utm_term is used as the Campaign keyword. You can learn more about Tracking Campaigns in Matomo or head over to the URL Builder.

I suggest to rephrase it to: By defaut Matomo supports the following Google Campaign UTM codes: utm_campaign , utm_source , utm_medium , utm_term , utm_content , utm_id. You can learn more about Tracking Campaigns in Matomo or head over to the URL Builder.

kr

paul gerhard

**Comment by michalkleiner** [comment]
Hi @IEEEPaulGerhard and thanks for the suggestion. There might be a slight misunderstanding about the list of params and which fields Matomo assigns them to, and then there's a wider support for more of them when using the MarketingCampaignsReporting plugin. I'll see if we can make the FAQ a bit clearer on that.
